src/instant-test/instant-test.component.js

```diff
@@ -4,34 +4,24 @@ import { LocalStorageContext } from "../shared/use-local-storage-data.hook";
 export default function InstantTest(props) {
   const params = new URLSearchParams(location.search);
   const isValid = params.has("name") && params.has("url");
-  const {
-    applications,
-    addApplication,
-    updateApplication,
-    resetApplications,
-  } = React.useContext(LocalStorageContext);
+  const { addApplication } = React.useContext(LocalStorageContext);
 
   React.useEffect(() => {
     if (!isValid) {
       return;
     }
 
-    resetApplications();
```

Comment:
Why was this removed? This is an important feature so that if you use the instant-test route twice that you don't accidentally keep around the previous app that you were testing which causes confusion when you're using it.

Reply:
Hmm I think I removed because since there is always only 1 application around, it will always override the last, so I thought there was no problem in removing this imperative reset! Hm... Now that I'm thinking, maybe I went overboard on deleting not-used-anymore code, and deprecated the multiple applications feature! Should I keep them, just in case this playground evolve again to support multiple apps?

Reply:
Ah good point. Yes this is unneeded now that there is only one app at a time. And no, I think only supporting one app at a time is the right way to go.